$NetBSD: patch-configure,v 1.1 2023/08/09 20:20:27 sjmulder Exp $

Depending on platform and compilation environment, GNU gettext
exports functions like bindtextdomain() as libintl_bindtextdomain() so
checking the symbol name directly doesn't work. When bindtextdomain()
isn't found help2man won't install localized man pages.

--- configure.orig	2022-12-13 00:48:03.000000000 +0000
+++ configure
@@ -3427,14 +3427,12 @@ else $as_nop
 cat confdefs.h - <<_ACEOF >conftest.$ac_ext
 /* end confdefs.h.  */
 
-/* Override any GCC internal prototype to avoid an error.
-   Use char because int might match the return type of a GCC
-   builtin and then its argument prototype would still apply.  */
-char bindtextdomain ();
+#include <libintl.h>
+
 int
 main (void)
 {
-return bindtextdomain ();
+bindtextdomain ("", "");
   ;
   return 0;
 }
